src/NHSD.BuyingCatalogue.Ordering.Api/Services/CreateOrder/CreateOrderRequest.cs

```diff
@@ -2,9 +2,10 @@
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.Services.CreateOrder
 {
-    public class CreateOrderRequest
+    sealed public class CreateOrderRequest

```

Comment:
To keep consistency with the rest of the API's can we stick with... `public sealed`?

Reply:
Yes! Dont know why I did that